charts/tempo-distributed/templates/gateway/secret-gateway.yaml

```diff
@@ -1,13 +1,14 @@
+{{- $dict := dict "ctx" . "component" "gateway" -}}
 {{- with .Values.gateway }}
 {{- if and .enabled .basicAuth.enabled (not .basicAuth.existingSecret) }}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "tempo.resourceName" (dict "ctx" . "component" "gateway") }}
+  name: {{ include "tempo.resourceName" $dict }}
   labels:
-    {{- include "tempo.labels" (dict "ctx" . "component" "gateway") | nindent 4 }}
+    {{- include "tempo.labels" $dict | nindent 4 }}
 stringData:
   .htpasswd: |
-    {{- tpl .basicAuth.htpasswd $ | nindent 4 }}
+    {{- tpl .basicAuth.htpasswd $dict.ctx | nindent 4 }}
```

Comment:
I take it that we need `$dict.ctx` here because the template doesn't support passing the dict, correct?

Reply:
Yes, that's not necessary... In fact $ and $dict.ctx are the same context.